tests/utils.py

```diff
@@ -226,8 +225,6 @@ def get_simple_test_pod(v1=False) -> V1Pod:
     pod = test_pod_json()
     if v1:
         annotations = pod['metadata']['annotations']
-        annotations[ANNOTATION_KEY_POD_SPEC_VERSION] = "1"
-        annotations[V1_ANNOTATION_KEY_JOB_ID] = annotations[ANNOTATION_KEY_JOB_ID]
 
         del annotations[ANNOTATION_KEY_JOB_ID]
```

Comment:
This does not reflect reality. A v1 pod *does* have this annotation

Reply:
I don't understand what's in a v1 pod and what isn't anymore.  We really only care about the _label_ now.  What would you like to see?

Reply:
I would like to see both of these lines gone (you already got line 230)
``` 
         annotations[V1_ANNOTATION_KEY_JOB_ID] = annotations[ANNOTATION_KEY_JOB_ID]

        del annotations[ANNOTATION_KEY_JOB_ID]
```